app/assets/javascripts/modules/feedback.js

```diff
@@ -9,41 +9,76 @@
   GOVUK.Modules.Feedback = function () {
     this.start = function (element) {
```

Comment:
I think the namespace should be ‘App’, not GOVUK

Reply:
The `GOVUK.Modules` script appears to look for Modules with either `find` or `start` methods.

Reply:
Here's accordion using init?
https://github.com/alphagov/govuk-frontend/blob/master/src/govuk/components/accordion/accordion.js